src/test/java/org/folio/edge/sip2/parser/BlockPatronMessageParserTests.java

```diff
@@ -27,8 +31,8 @@ void testParse() {
         + "AApatron_id|AC|AOuniversity_id|");
 
     assertEquals(FALSE, blockPatron.getCardRetained());
-    assertEquals(transactionDate.getOffset(),
-        blockPatron.getTransactionDate().getOffset());
+    assertEquals(transactionDate.withOffsetSameInstant(ZoneOffset.UTC),
```

Comment:
While I don't think we need to change this now (or ever really), `transactionDate` is already UTC as defined on line 24, so the `withOffsetSameInstant(ZoneOffset.UTC)` shouldn't do anything. It makes the test a little more complicated, but I think it is a big no-op. This is repeated throughout the other parser tests.